lib/topics_filter.rb

```diff
@@ -332,4 +352,8 @@ def include_topics_with_all_tags(tag_ids)
   def include_topics_with_any_tags(tag_ids)
     @scope = @scope.joins(:topic_tags).where("topic_tags.tag_id IN (?)", tag_ids).distinct(:id)
   end
+
+  def integer_string?(string)
+    string =~ /^\d+$/
```

Comment:
Probably doesn't matter, but just to be extra sure, shall we use \A \z here?

Reply:
Done :+1